docs/blog/posts/2024/srlinux-asymmetric-routing.md

```diff
@@ -0,0 +1,1801 @@
+---
+date: 2024-10-14
+tags:
+    - bgp
+    - evpn
+authors:
+    - aninda
+---
+
+# Asymmetric routing with SR Linux in EVPN VXLAN fabrics
+
+This post dives deeper into the asymmetric routing model on SR Linux.  
+The topology in use is a 3-stage Clos fabric with BGP EVPN and VXLAN, with
```

Comment:
I think it would be great to provide some intro here.
Maybe move it from the `Reviewing the asymmetric routing model` section.

e.g. why the two modes exist, what are the differences between them.

you can use this info
<img width="1681" alt="image" src="https://github.com/user-attachments/assets/e226b0dc-cab9-4c82-a26e-75a28ba6efb8">

Reply:
keeping this out for now as I feel it bloats the post and adds more questions in the minds of the reader.